### PR TITLE
Create RTL css  for tabs, delete inline css

### PR DESCRIPTION
**Pull request by eyalvo-matrix**
Hi all, 
This commit purpose is to add RTL support for the tabs mod
in the commit :
1. Delete inline css
2. create  rtl_tabs_style_full.css file

Below added by bradymiller:
Can demo this PR here: http://www.open-emr.org/wiki/index.php/Development_Demo#Gamma_-_Up_For_Grabs_Demo

### CODE REVIEW

**Comment by bradymiller** [comment]
Added up for grabs demo for this here:
Can demo this PR here: http://www.open-emr.org/wiki/index.php/Development_Demo#Gamma_-_Up_For_Grabs_Demo

File: interface/themes/tabs_style_full.css [hunk 1]
```diff
@@ -157,6 +164,15 @@ body
...
+
+#attendantData .patientDataColumn:nth-child(1) span{
+    float:left;
+}
+
+#attendantData .patientDataColumn:nth-child(1) .patientInfo:nth-child(2) {
+    float: left;
+}
+
 .patientInfo a {
```
Comment (bradymiller) [inline interface/themes/tabs_style_full.css]: Also need to add these changes to interface/themes/tabs_style_compact.css

File: interface/themes/rtl_tabs_style_full.css [hunk 2]
```diff
@@ -0,0 +1,468 @@
+/**
```
Comment (bradymiller) [inline interface/themes/rtl_tabs_style_full.css]: Wouldn't it make much more sense to follow the strategy you used for the main theme to also provide rtl for the tab themes (note your group did that awhile back)?
Check out that mechanism and I would be happy to provide guidance/pointers as needed.

Comment (matrix-amiel) [reply interface/themes/rtl_tabs_style_full.css]: Hi @bradymiller .
I preferred to continue with the new logic of the load assets that enables to add option for RTL languages in the config.yaml file and not with the logic of  'main theme' in the globals.php. I think it's cleaner and more simple.

File: interface/themes/tabs_style_full.css (hunk 1 above)
Comment (matrix-amiel) [reply interface/themes/tabs_style_full.css]: Done.

File: interface/themes/rtl_tabs_style_full.css (hunk 2 above)
Comment (bradymiller) [reply interface/themes/rtl_tabs_style_full.css]: Agreed :)

**Comment by bradymiller** [comment]
hi @eyalvo-matrix , Testing well and will bring into codebase. Thanks for the fixes! -brady

**Comment by bradymiller** [comment]
hi @eyalvo-matrix ,

Of course right after I brought this in, I noted the following bug that this code brought in (in non-rtl the patient name and id is getting pushed to the left):
image

Can you address this?

thanks,
-brady

**Comment by matrix-amiel** [comment]
O.k. Sorry.
I'm fixing it now.

**Pull request merged**